resolwe_bio/docker_images/dnaseq/packages-manual/bokeh-assets.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+. /var/cache/build/packages-manual/common.sh
+
+download_and_verify \
+    bokeh \
+    assets \
+    0.12.9 \
+    1a47bc2a7be7a1fea12d449a694a7efb50a1b1b879cb1c1d8ec9412d13b988ae \
+    https://cdn.pydata.org/bokeh/release/bokeh-\${version}.min.css \
```

Comment:
The trailing backslash in the last line of the command is redundant. Remove it also in at the end of this file.